lug/config.yaml

```diff
@@ -14,10 +14,162 @@ dummy:
     retry_interval: 60
```

Comment:
Please remove `.DS_Store` files and add them to .gitignore

Reply:
fixed